Utilities/bootstrap

```diff
@@ -593,30 +620,38 @@ def add_rpath_for_cmake_build(args, rpath):
     note(' '.join(add_rpath_cmd))
     subprocess.call(add_rpath_cmd, stderr=subprocess.PIPE)
 
+def build_packagedescription_libs_with_cmake(args):
+    """Builds the PackageDescription libraries using CMake."""
+    note("Building PackageDescription libraries (with CMake)")
+
+    cmake_flags = ["-DFIND_PM_DEPS:BOOL=NO"]
+    targets = ["PD4", "PD4_2", "PackagePlugin"]
```

Comment:
I think this is probably the only thing that needs to change after https://github.com/apple/swift-package-manager/pull/3464 is merged.